build.gradle.kts

```diff
@@ -1,7 +1,8 @@
 subprojects {
     group = "com.utopia-rise"
-    version = "0.1.0"
+    version = "0.1.0" //remember to bump this version in conjunction with the version in the CompilerPluginConst
```

Comment:
Something for later, but we can make this so that we only need to manage a single version. The idea here is that `CompilerPluginConst` will load the version value from a `build.properties` file included in the jar. `build.properties`. The properties file is then processed by gradle via the standard task `processResources`. Relevant sources from my binding:

[1] https://github.com/raniejade/godot-kotlin/blob/master/godot-kotlin-gradle-plugin/src/main/resources/build.properties
[2] https://github.com/raniejade/godot-kotlin/blob/master/godot-kotlin-gradle-plugin/build.gradle.kts#L55
[3] https://github.com/raniejade/godot-kotlin/blob/master/godot-kotlin-gradle-plugin/src/main/kotlin/godot/gradle/buildProperties.kt#L6

Reply:
I looked for an elegant solution but this didn't came to my mind... I really like it. As i have to wait for MpApt to merge. I'll include this in this PR as well. This is billiant!